core/src/main/java/it/pagopa/selfcare/dashboard/core/InstitutionServiceImpl.java

```diff
@@ -366,7 +367,10 @@ public OnboardingRequestInfo getOnboardingRequestInfo(String tokenId) {
         log.trace("getOnboardingRequestInfo start");
         log.debug("getOnboardingRequestInfo tokenId = {}", tokenId);
         final OnboardingRequestInfo onboardingRequestInfo = partyConnector.getOnboardingRequestInfo(tokenId);
-        onboardingRequestInfo.getManager().setUser(userRegistryConnector.getUserByInternalId(onboardingRequestInfo.getManager().getId(), USER_FIELD_LIST_ENHANCED));
+        // In case of PT onboarding, the field manager is empty
```

Comment:
you could add this information on open api, for example "it is required when institution type is not PT", at https://github.com/pagopa/selfcare-dashboard-backend/blob/2223731d777d44f495445a213e50b21175e5530f/app/src/main/resources/swagger/api-docs.json#L3991